src/skmultiflow/anomaly_detection/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""
+The :mod:`skmultiflow.anomaly_detection` module includes anomaly detection methods based.
+"""
```

Comment:
There is a typo, suggested text:

```
Please restore the parameter in the docstring since it is used to generate the documentation. What I mean to say is that it should be stated that the parameter is in the signature but not used, sorry for not being clear.

Suggestion:
```
The :mod:`skmultiflow.anomaly_detection` module includes anomaly detection methods.
```

Reply:
> Thanks for the last changes. The code seems ready, there are just minor documentation changes left that I highlighted.
> 
> Please use a normal commit (not a forced one) to upload the new versions. The reason for using a normal commit is to try to get `codecov` to recognize the change and provide the coverage report for the entire project, not only for your changes ;-)

I will keep that in mind :) By the way, I have added the documentation changes.